config/prow/cluster/kubernetes_external_secrets.yaml

```diff
@@ -39,3 +39,16 @@ spec:
   - key: gke_k8s-prow_us-central1-f_prow__prow-monitoring__grafana
     name: password
     version: latest
+---
+apiVersion: kubernetes-client.io/v1
+kind: ExternalSecret
+metadata:
+  name: snyk_token
+  namespace: prow-test
```

Comment:
is this correct namespace where periodic prow jobs are run ?  (https://github.com/kubernetes/test-infra/pull/22293)

Reply:
A: `test-pods` (also populated in env var `PROWJOB_POD_NAMESPACE`)
see https://github.com/kubernetes/k8s.io/pull/2078